views/js/runner/testStore.js

```diff
@@ -290,6 +302,38 @@ define([
                 });
             },
 
+            /**
+             * Observe changes on the given store
+             *
+             * @param {String} storeName - the name of the store to observe
+             * @returns {testStore} chains
+             */
+            trackChanges : function trackChanges(storeName){
+                changes[storeName] = false;
+                return this;
+            },
+
+            /**
+             * Has the store some changes
+             *
+             * @param {String} storeName - the name of the store to set as volatile
+             * @returns {Boolean} true if the given store has some changes
+             */
+            hasChanges : function hasChanges(storeName){
+                return changes[storeName] === true;
+            },
+
+            /**
+             * Reset the change listening
+             *
+             * @param {String} storeName - the name of the store
+             * @returns {testStore} chains
+             */
+            resetChanges : function resetChanges(storeName){
```

Comment:
why do you need both: `resetChanges` and `trackChanges` which do the same things?

Reply:
Also, resetChanges won't reset real changes, only state of changes